source/gui/settingsDialogs.py

```diff
@@ -530,6 +542,14 @@ def makeSettings(self, settingsSizer):
 		self.beepForCapsCheckBox = wx.CheckBox(self, wx.NewId(), label = _("&Beep for capitals"))
 		self.beepForCapsCheckBox.SetValue(config.conf["speech"][getSynth().name]["beepForCapitals"])
 		settingsSizer.Add(self.beepForCapsCheckBox,border=10,flag=wx.BOTTOM)
+		speakDigitsSizer = wx.BoxSizer(wx.HORIZONTAL)
+		# Translators: Label for a combo box for setting how nvda speaks numbers.
+		digitsLabel = wx.StaticText(self, wx.ID_ANY, label=_("Speak &Numbers as"))

```

Comment:
Well, given the added information to the user guide regarding the 
different behaviour across synthesizers, I think it is ok as it is.